### ISSUE TITLE
Java client retrieveRefreshTokenById builds a wrong request URL

### ISSUE THREAD

**Issue by dmitryzan** [FusionAuth/fusionauth-issues#1724]
### Description 
`FusionAuthClient.retrieveRefreshTokenById(userId)` builds a wrong request not matching the docs.
Currently:
```Java
return start(RefreshTokenResponse.class, Errors.class)
    .uri("/api/jwt/refresh")
    .urlSegment(userId)
    .get()
    .go();
```
should be `GET /api/jwt/refresh?userId={userId}`:
```Java
return start(RefreshTokenResponse.class, Errors.class)
    .uri("/api/jwt/refresh")
    .urlParameter("userId", userId)
    .get()
    .go();
```

### Affects versions
Library version `io.fusionauth:fusionauth-java-client:1.36.0` 

### Steps to reproduce
1. Call `FusionAuthClient.retrieveRefreshTokenById(userId)`
2. See 404

### Expected behavior
200 response with a list of refresh tokens.

**Comment by mooreds** [comment]
Thanks for filing this issue! 

I think there are two things going on.

The first is that `retrieveRefreshTokenById` is not documented correctly. This actually retrieves a single refresh token by its id. It doesn't return all refresh tokens for a user. 

In addition, the parameter name is incorrect for `retrieveRefreshTokenById`. It should be `tokenId`.

I think the call you want is `retrieveRefreshTokens` which does put the `userId` in the URL parameter list. Can you please try that method and let me know if it works for you?

**Comment by dmitryzan** [comment]
Thank you for clarification!
The `retrieveRefreshTokens(UUID userId)` works!
Agreed, it was a confusion looking at the `retrieveRefreshTokenById(UUID userId)` parameter name/description. For some reason missed the other method.

**Comment by robotdan** [comment]
Moving to 1.38.0. We will be re-building our client libraries based upon our OpenAPI spec, and won't be releasing our now "internal" Java REST client at the release of 1.37.0.

**Comment by robotdan** [comment]
Looks like this was fixed in 1.37.0
